Core/Frameworks/BaikalAdmin/Controller/Install/VersionUpgrade.php

```diff
@@ -365,7 +365,7 @@ protected function upgrade($databaseConfig, $sVersionFrom, $sVersionTo) {
             }
         }
         if (version_compare($sVersionFrom, '0.5.1', '<')) {
-            if ($databaseConfig['mysql'] === false) {
+            if ($databaseConfig['backend'] == 'sqlite') {
```

Comment:
Same here, keep the old variant (and in 2 other places in this file)

Reply:
Fixed in 35a3d01